kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -254,7 +254,7 @@ define([
      * @returns {boolean} true|false
      */
     function isValidBackendJobStateObject(backendJobState) {
-        const required = ['jobState'];
+        const required = ['job_id', 'jobState'];
```

Comment:
this change was made in the backend in the previous PR; this adds an extra bit of validation in the frontend.